snuba/query/snql/__init__.py

```diff
@@ -0,0 +1,13 @@
+from typing import Any, MutableMapping
+from snuba.datasets.dataset import Dataset
+from snuba.query.logical import Query
+
+
+def parse_snql_query(body: MutableMapping[str, Any], dataset: Dataset) -> Query:
```

Comment:
The body in snql is just a string. And you can just pass an empty dictionary to the Query   constructor since that is the old query representation we do not support.